charts/victoria-metrics-anomaly/values.yaml

```diff
@@ -7,10 +7,12 @@ global:
       adaptSecurityContext: "auto"
       
 image:
+  # -- Victoria Metrics anomaly Docker registry
+  registry: ""
   # -- Victoria Metrics anomaly Docker repository and image name
-  repository: docker.io/victoriametrics/vmanomaly
+  repository: victoriametrics/vmanomaly
   # -- Tag of Docker image
-  tag: "" # rewrites Chart.AppVersion
+  tag: v1.13.3 # rewrites Chart.AppVersion
```

Comment:
As far as I can see, charts are not specifying `.image.tag` anymore and using `.Chart.AppVersion` instead. Is this an intentional change?

Reply:
vmanomaly tagging policy differs from other apps. despite it's an enterprise product, but it has no `-enterprise` suffix in a tag name. I've recently added a PR to vmanomaly repo to fix it
https://github.com/VictoriaMetrics/vmanomaly/pull/243

Reply:
Thanks for clarification. Could you update a tag to `v1.15.4` then?
That's the latest version based on [changelog](https://docs.victoriametrics.com/anomaly-detection/changelog/)

Reply:
[code in a common template](https://github.com/VictoriaMetrics/helm-charts/blob/0735f3ce0e7a74fa384d0d93bb3123a55cec8b16/charts/victoria-metrics-common/templates/_image.tpl#L7) checks if `image.tag` wasn't set and for enterprise products (which have license.key, license.secret.name or eula) it sets `image.variant` to `enterprise`